src/components/scatterplot/Scatterplot.js

```diff
@@ -1,6 +1,10 @@
+import React from 'react';
+
 import { SelectableScatterplotLayer } from '../../layers';
 import { cellLayerDefaultProps, DEFAULT_COLOR } from '../utils';
+import CellEmphasisSubscriber from '../CellEmphasisSubscriber';
 import AbstractSelectableComponent from '../AbstractSelectableComponent';
+import StatusTooltipSubscriber from '../StatusTooltipSubscriber';
```

Comment:
We want to avoid dependencies on the `*Subscriber` classes in the non-`*Subscriber` classes.

Reply:
I have now done this with wrapper components that begin with the word `Hoverable`. Should they instead be named with a suffix rather than this prefix, like `ScatterplotSubscriberWithTooltip` or something?